clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/validators/OpenstackAttributeValidator.groovy

```diff
@@ -113,11 +130,82 @@ class OpenstackAttributeValidator {
     if (result) {
       def openstackCredentials = accountCredentialsProvider.getCredentials(account)
       if (!(openstackCredentials?.credentials instanceof OpenstackCredentials)) {
-        errors.rejectValue("${context}.account",  "${context}.account.notFound")
+        errors.rejectValue("${context}.account", "${context}.account.notFound")
         result = false
       }
     }
     result
   }
 
+  /**
+   * Validate string is in UUID format.
+   * @param value
+   * @param attribute
+   * @return
+   */
+  def validateUUID(String value, String attribute) {
+    boolean result = validateNotEmpty(value, attribute)
+    if (value) {
+      try {
+        UUID.fromString(value)
+      } catch (IllegalArgumentException e) {
+        errors.rejectValue("${context}.${attribute}", "${context}.${attribute}.notUUID")
+        result = false
+      }
+    }
+    result
+  }
+
+  /**
+   * Validate integer is an HTTP status code
+   * @param value
+   * @param attribute
+   * @return
+   */
+  def validateHttpStatusCode(Integer value, String attribute) {
+    boolean result = validateNotEmpty(value, attribute)
+    if (result) {
+      try {
+        HttpStatus.valueOf(value)
+      } catch (IllegalArgumentException e) {
+        reject(attribute, 'invalid Http Status Code')
```

Comment:
The other validators use `rejectValue` instead of `reject`. Is there a reason for the difference?